MAUI/Rating/Appearance-Customization.md

```diff
@@ -0,0 +1,213 @@
+---
+
+layout: post
+title: Appearance Customization in .NET MAUI Rating control | Syncfusion
+description: Learn here all about Appearance Customization support in Syncfusion .NET MAUI Rating (SfRating) control and more.
+platform: maui
+control: Rating
+documentation: ug
+
+---
+
+# Appearance Customization in .NET MAUI Rating (SfRating)
+
+We can customize the rated and unrated fill color, stroke and stroke thickness using the following properties of [`RatingSettings`].
+
+* [`RatedFill`]
+* [`UnratedFill`]
+* [`RatedStroke`]
+* [`UnratedStroke`]
+* [`RatedStrokeThickness`]
+* [`UnratedStrokeThickness`]
+ 
+## Set Fill Color
+
+[`SfRating`] control has support to set the fill color for the rated and unrated items.
+
+### Rated Items
+
+The [`RatedFill`] property fills the rated item with the specified solid color in the [`SfRating`]
```

Comment:
The [`RatedFill`] property fills the rated items with the specified solid color in the [`SfRating`].

Reply:
Changed